chain/src/Pos/Chain/Block/Header.hs

```diff
@@ -427,6 +432,17 @@ mkMainHeaderExplicit pm prevHash difficulty slotId sk pske body extra =
                (BlockSignature $ sign pm SignMainBlock sk toSign)
                (makeSignature toSign)
                pske
+    -- If `pske :: ProxySKBlockInfo` is Just, indicating delegation, we use
+    -- the PublicKey of the delegator (in Original era, this is the Stakeholder
+    -- whose stake was selected by FTS. In OBFT era, this is the Genesis
+    -- Stakeholder, selected via round-robin. In both cases, the delegator is
+    -- delegating the right to mint a block to the delegatee).
+    --
+    -- If `pske :: ProxySKBlockInfo` is Nothing, we use `toPublic` of the `pk`
```

Comment:
@mhuesch: We mean to say, "we use `toPublic` of the `sk`", right?

Reply:
Agh, yes, good catch.

Reply:
I just added a commit to fix the typo. I didn't squash / force push because I don't want to upset any work than anyone else has based on the branch, but it would be good to do that before merging.